client/state/index.js

```diff
@@ -111,7 +112,9 @@ export const reducer = combineReducers( {
 	ui,
 	users,
 	wordads,
-} );
+};
+
+export const reducer = combineReducers( { ...extensions.reducers(), ...reducers } );
```

Comment:
>  Ideally, we'd namespace all extensions

Is there anything stopping us from doing this now? I would think we'd want this in before it's copy pasted anywhere else.